boolean/test_boolean.py

```diff
@@ -180,9 +249,9 @@ def test_eval(self):
         a = boolean.Symbol("a")
         self.assertTrue(~a == ~a)
         self.assertFalse(a == boolean.parse("~~a", eval=False))
-        self.assertTrue(a == ~~a)
-        self.assertTrue(~a, ~~~a)
-        self.assertTrue(a, ~~~~a)
+        self.assertTrue(a == ~ ~a)
```

Comment:
Can this still receive strings such as `"~~~a"` or does it need spaces there? ...'cause I'd much rather like it to support it.